gtsfm/loader/loader_base.py

```diff
@@ -204,14 +204,26 @@ def get_relative_pose_prior(self, i1: int, i2: int) -> Optional[PosePrior]:
         """Get the prior on the relative pose i2Ti1
 
         Args:
-            i1 (int): _description_
-            i2 (int): _description_
+            i1 (int): index of first image
+            i2 (int): index of second image
 
         Returns:
             Pose prior, if there is one.
         """
         return None
 
+    def get_relative_pose_priors(self, pairs: List[Tuple[int, int]]) -> Dict[Tuple[int, int], Optional[PosePrior]]:
+        """Get *all* relative pose priors for i2Ti1
+
+        Args:
+            pairs: all (i1,i2) pairs of image pairs
+
+        Returns:
+            A dictionary of PosePriors (or None) for all pairs.
+        """
+
+        return {pair: self.get_relative_pose_prior(*pair) for pair in pairs}
```

Comment:
Maybe we need to filter out None just as in Hilti loader

Reply:
Yes we do.

Reply:
Done